### ISSUE TITLE
Docs for affects-mails outdated?

### ISSUE THREAD

**Issue by MrChick** [tenancy/docs#166]
I tried using affects-mails according to the example in the docs today.
Didn't work as expected, got an error that my mailer didn't have a setFrom() method.
I checked, the Mailer associated with my ConfigureMails event is an Illuminate\Mail\Mailer, not a Swift_Mailer.

Is the example outdated or did I do something wrong?

(tenancy/affects-mails, Laravel 7)

**Comment by sniper7kills** [comment]
Sorry for the delay in a reply; the team has been super busy.

Unfortunately, I'm not overly familiar w/ the affects-mails module and those who are have been busy.

It is very possible that the docs are in-correct and we are planning some major improvements to the module in the future.

**Comment by MrChick** [comment]
Yeah no worries as I got it working. Was just asking because I stumbled over it. If someone can confirm that this is in fact the case I'd submit a quick update for the docs.

**Comment by sniper7kills** [comment]
Oh, we'd love a PR if you would be willing to make one!
